switchapi/switch_nhop_int.h

```diff
@@ -75,12 +75,8 @@ extern "C" {
   })
 
 #define switch_nhop_get_group(_device, _handle, _info)                    \
-  ({                                                                \
-    switch_nhop_info_t *_tmp_nhop_info = NULL;                      \
-    (void)(_tmp_nhop_info == *_info);                               \
```

Comment:
Other places in the same file has similar check's. Shall we merge the current changes and raise a new patch for other occurances?

Reply:
My inclination is to address only this occurrence in the current changelist and address other issues in a subsequent commit. I see a number of other potential improvements that could be made at that time.